iOS Example Frame Carthage/Cartfile

```diff
@@ -0,0 +1,3 @@
+
+github "Alamofire/Alamofire" ~> 5.4.0
+github "marmelroy/PhoneNumberKit" ~> 3.3.3
```

Comment:
Should we update our Podfile to ~> 3.3.3 too so all the sample integrations are in sync?

Reply:
I think we should downgrade to  ~> 3.3.0 to allow a wider range of versions, just for compatibility